rules/do-you-know-how-modern-stateless-auth-works/rule.md

```diff
@@ -0,0 +1,75 @@
+---
+type: rule
+title: Do you know how Modern Stateless Authentication works?
+uri: modern-stateless-authentication
+authors:
+  - title: Dhruv Mathur
+    url: https://www.ssw.com.au/people/dhruv-mathur/
+  - title: Gordon Beeming
+    url: https://www.ssw.com.au/people/gordon-beeming/
+  - title: Matt Goldman
+    url: https://www.ssw.com.au/people/matt-goldman/
+created: 2024-03-06T13:42:55.753Z
+guid: fa645fa6-4054-448b-9b4b-d60a66422f1f
+---
+
+# Do you know how Modern Stateless Authentication works?
```

Comment:
This is a duplicate of the title

- [ ] 1. Please remove

Reply:
>1. Please remove

✅Done - removed.